lib/onfleet-ruby.rb

```diff
@@ -47,7 +47,7 @@ def self.request api_url, method, params={}
     raise AuthenticationError.new("Set your API Key using Onfleet.api_key = <API_KEY>") unless @api_key
 
     begin
-      response = RestClient::Request.execute(method: method, url: self.base_url+api_url, payload: params.to_json, headers: self.request_headers)
+      response = RestClient::Request.execute(method: method, url: self.base_url+api_url, payload: params.to_json, headers: self.request_headers, content_type: :json)
```

Comment:
Probably we don't need both headers and content_type here. It's enough to add content_type to request_headers below.

Reply:
But thanks for such PR, @davidvetrano! We began having problems with content-type today on our project.

Reply:
@AlexBelov That would make sense. I didn't dig into rest-client to verify that the behavior is identical without explicitly specifying the content-type on the request but I'd imagine that setting it again has no effect. We recently started enforcing the content-type for requests to the Onfleet API to only accept JSON across the board so I figured I'd help out here. I'll look into any differential behavior and tweak as needed on Monday.